components/empty-state/demo/empty-state.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta name="viewport" content="width=device-width, initial-scale=1.0">
+		<meta charset="UTF-8">
+		<link rel="stylesheet" href="../../demo/styles.css" type="text/css">
+		<script type="module">
+			import '../../demo/demo-page.js';
+			import '../../demo/demo-snippet.js';
```

Comment:
We don't often do this, but the demo-page automatically imports the demo-snippet so that demo pages don't have to. It does it purely as a convenience.